programs/programs/nc/nc_crisis_intervention/calculator.py

```diff
@@ -0,0 +1,51 @@
+from programs.programs.calc import MemberEligibility, ProgramCalculator, Eligibility
+import programs.programs.messages as messages
+
+
+class CrisisIntervention(ProgramCalculator):
+    expenses = ["rent", "mortgage", "heating"]
+    fpl_percent = 1.5
+    percent_of_fpl = 4
+    # resource_limit = 2250
```

Comment:
You don't use `expenses` or `resource_limit`. Also, what is the difference between `fpl_percent` and `percent_of_fpl`? Could you rename one of them so that it is clearer what they do?
```suggestion
    fpl_percent = 1.5
    percent_of_fpl = 4
```